neigh_linux.go

```diff
@@ -235,7 +238,7 @@ func (h *Handle) neighList(linkIndex, family, flags int) ([]Neigh, error) {
 	return res, nil
 }
 
-func NeighDeserialize(m []byte) (*Neigh, error) {
+func DeserializeNeigh(m []byte) (*Neigh, error) {
```

Comment:
This has been an exported netlink package method.
We should not rename it for backward compatibility of existing code which may use it.

I like it too for the methods names to be consistent.
If you want to stick to it, you could 
```
func NeighDeserialize(m []byte) (*Neigh, error) {
+        return DeserializeNeigh(m)
+}
+func DeserializeNeigh(m []byte) (*Neigh, error) {
```

Reply:
Yep good point, I'll add the renamed methods to keep compatibility.